golangcilint.yaml

```diff
@@ -1,11 +1,38 @@
 run:
-  deadline: 60m
-  skip-dirs:
-    - "internal"
+  timeout: 60m
 
 linters:
   enable-all: true
-  disable: funlen,gocognit,exhaustivestruct,paralleltest,forbidigo,ireturn,wrapcheck,exhaustive,maintidx,exhaustruct,nosnakecase,interfacer,structcheck,varcheck,deadcode,ifshort,golint,scopelint,maligned,rowserrcheck,sqlclosecheck,revive,musttag,depguard
+  disable:
+    - funlen
+    - gocognit
+    - paralleltest
+    - forbidigo
+    - ireturn
+    - wrapcheck
+    - exhaustive
+    - maintidx
+    - exhaustruct
+    - rowserrcheck
+    - sqlclosecheck
+    - revive
+    - musttag
+    - depguard
+    - execinquery
+    - gomnd
+    - inamedparam
+    - protogetter
+    - promlinter
+    - perfsprint
+    - testifylint
```

Comment:
I think we can simply remove the dependency on github.com/stretchr/testify/assert instead. Seems to be used in a single case